GpsDataCapturer/wear/src/main/java/com/google/gpsdatacapturer/WearGpsMainActivity.java

```diff
@@ -199,6 +208,30 @@ public void onServiceConnected(ComponentName name, IBinder service) {
             //get the gpsDataCaptureService
             gpsDataCaptureService =
                     ((GpsDataCaptureService.GpsDataCaptureBinder) service).getService();
+
+            //start capture via intent
+            if(adbIntent != null) {
+                //Extra the location api type
+                if(adbIntent.getExtras() != null){
+                    Log.d(TAG, "Intent extra is " + adbIntent.getExtras().toString());
+                    boolean type_from_intent = adbIntent.getBooleanExtra("fused_location_type", false);
+                    Log.d(TAG, "fused_location_type: " + type_from_intent);
+                    if(type_from_intent){
+                        locationApiType = LocationApiType.FUSEDLOCATIONPROVIDERCLIENT;
+                    }
+                    Log.d(TAG, "LocationApiType: " + locationApiType);
+                    gpsDataCaptureService.setLocationApiType(locationApiType);
+                }
+                //Start capture if the action is START_CAPTURE
+                if (adbIntent.getAction() != null) {
+                    Log.d(TAG, "Intent action: " + adbIntent.getAction());
+                    if (adbIntent.getAction().equals(
+                            "com.google.gpsdatacapturer.START_CAPTURE")) {
+                        Log.d(TAG, "Start capture via intent");
+                        gpsDataCaptureService.startCapture();
+                    }
+                }
+            }
```

Comment:
Please see similar comments on phone activity

Reply:
Will rename it to `launchIntent`